schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql

```diff
@@ -0,0 +1,3 @@
+drop function if exists pxt_session_cleanup
```

Comment:
Please move this file to 4.2.7-to-4.2.8, as I had to tag for submitting Beta1.

Reply:
And fix the conflicts for the spacewalk-java changelog :-)

Reply:
@juliogonzalez I've now moved it to `4.2.7-to-4.2.8`

Reply:
Thanks. It's merged now :-)